nx.json

```diff
@@ -85,5 +86,35 @@
         "linter": "eslint"
       }
     }
+  },
+  "release": {
+    "projects": [
+      "packages/*",
+      "!packages/plugin-lighthouse",
+      "!packages/nx-plugin"
+    ],
+    "projectsRelationship": "fixed",
+    "changelog": {
+      "automaticFromRef": true,
+      "projectChangelogs": false,
+      "workspaceChangelog": {
+        "createRelease": "github"
+      }
+    },
+    "git": {
+      "commit": true,
+      "push": true,
+      "pushRemote": "origin",
+      "tag": true,
+      "commitMessage": "release: {version} [skip ci]",
+      "commitArgs": "--no-verify"
+    },
+    "version": {
+      "conventionalCommits": true,
+      "generatorOptions": {
+        "skipLockFileUpdate": true
```

Comment:
`skipLockFileUpdate` is an opinionated choice as I think that updating the lock file might have (bad) unattended side effects. 
For example, messing up with your NPM cache or trigger false change detection in the CI

Reply:
OK, thanks for the explanation. One small annoyance we have with `jscutlery/semver` is that it only updates the `version` in `package.json` and not `package-lock.json`, so running `npm install` (e.g. by running E2E tests locally) will often edit the lock file. But we'll see how it goes with Nx release, we can always tweak it later :thinking:

Reply:
If you follow [this approach](https://nx.dev/recipes/nx-release/update-local-registry-setup#the-updated-version) to replace publishing with nx release library you can customize the release option including the npm registry and the lock file update.